### DISCUSSION TITLE
Invidious

### DISCUSSION THREAD

**Discussion by sqrvrt** [catppuccin/catppuccin#1986] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

It seems website doesn't support direct theming so it should be done with custom css.

### Any additional comments?

Invidious is an open source youtube frontend.

[Invidious official website](https://invidious.io)
[Invidious instances list](https://invidio.us)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by rubyowo** [comment]
**what**

**Comment by rubyowo** [comment]
we already have an [invidious port](https://github.com/catppuccin/invidious).
Please check if a port already exists before making an issue. Thanks.

**Comment by spookyintheam** [comment]
Going to close this since we already have an invidious port.
